dali/python/nvidia/dali/ops.py

```diff
@@ -404,21 +404,25 @@ def __init__(self, **kwargs):
             self._spec.AddArg("preserve", self._preserve)
             self._preserve = self._preserve or self._schema.IsNoPrune()
 
-            if "output_dtype" in kwargs.keys():
-                if "dtype" in kwargs.keys():
-                    raise TypeError(
-                        ("Operator {} got an unexpected 'output_dtype' argument when " +
-                         "'dtype' is already provided").format(type(self).__name__))
-                # show only this warning
+            # Check for any deprecated arguments that should be replaced or removed
+            arg_names = list(kwargs.keys())
+            for arg_name in arg_names:
+                if not self._schema.IsDeprecatedArg(arg_name):
+                    continue
+                msg = self._schema.DeprecatedArgMsg(arg_name)
+                new_name = self._schema.DeprecatedArgRenamedTo(arg_name)
+                ignore = self._schema.DeprecatedArgIgnore(arg_name)
+                if new_name:
+                    msg += " Use \"{}\" instead.".format(new_name)
```

Comment:
1. This will produce a leading space in case of empty message.
2. When the message does not end in full stop, it will look awkward (a new line would be better)
```suggestion
                    if msg: msg += "\n"
                    msg += "Use \"{}\" instead.".format(new_name)
```

Reply:
I am assuming that the message is not empty and terminated with ".". That being said, I'll add your suggestion